Ludwig timeseries models

surendra1472 opened the pull request:
Ludwig timeseries models

George3d6 commented:
Actually further note @surendra1472 running the examples locally, they seems not to work for me.
E.g. for `temperature_forecast` the names in the ludwig config are wrong and the file paything is wrong.

Are you certain that you've pushed the last version you had locally ?

surendra1472 commented:
Addressed all the pr comments as a new pr https://github.com/mindsdb/mindsdb-examples/pull/9
Since this pr raised from a forked branch(previously i did not have write access), so raised new pr from this repo itself.
